### PR TITLE
CoW Protocol: New Solver Addresses

### PR DESCRIPTION
**Pull request by nlordell**
This PR adds new CoW Protocol solver addresses to the spellbook.

@gentrexha and @bh2smith can confirm the addresses are valid (also please double check I didn't misname any of the solvers).

### CODE REVIEW

File: models/cow_protocol/ethereum/cow_protocol_ethereum_solvers.sql [hunk 1]
```diff
@@ -101,7 +101,36 @@ known_solver_metadata (address, environment, name) as (
...
+                 ('0x31Ac5E51a168B5179c703f3b05F120748C8c7c88', 'barn', 'Gnosis_BalancerSOR'),
+                 ('0xe374f64513f2432BcbF964B5ab84bD350D1FF222', 'barn', 'Otex'),
+                 ('0x3cdC01749eEEb4A26b1a9a9611328F232bE06be7', 'barn', 'PLM'),
+                 ('0xDE478f29C9566499f741cBF91CB068F1C2614B69', 'barn', 'Laertes'),
+                 ('0xD01BA5b3C4142F358EfFB4d6Cb44A11E31600330', 'barn', 'Seasolver'),
+                 ('0xF330Ee61d01ef1AC9FFA64662263e5E2DE93bbdE', 'barn', 'Quasilabs'),
+                 ('0x3C3513c88bD7A919Cb732F777Cd80cd773Beb011', 'barn', 'Raven'),
+                 ('0xAC0128ACC5c15945aB4E81F09f58CF05ec1844Ed', 'barn', 'Barter'),
+                 ('0xd3eEc78a89B04f95965927F408F07C13996B8378', 'barn', 'PropellerHeads'),
+                 ('0x452d604f08affFc4E87E74e3279BdBdeCeD07232', 'prod', 'PropellerHeads')
```
Comment (MartinquaXD) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_solvers.sql:133]: Should `prod PropellerHeads` be moved to the end of the `prod` list?

Comment (nlordell) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_solvers.sql]: Reorder the solvers in the list to be grouped like before.

**Comment by bh2smith** [comment]
I suppose we could lift the conditions of that test. Can you think of an alternative test we could be using in place of it? Also we should probably check if solvers even need to be active for rewards. I would imagine not. The only problem is if they have been unvouched for (which is totally different). So why not remove these solvers if they aren't being used? Then you could pass the test as is.

File: models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql [hunk 2]
```diff
@@ -63,4 +72,4 @@ select CAST(solver AS VARCHAR(42)) as address,
       case when name is not null then name else 'Uncatalogued' end      as name,
       active
 from registered_solvers
-    left outer join known_solver_metadata on CAST(solver AS VARCHAR(42)) = lower(address);
\ No newline at end of file
+    left outer join known_solver_metadata on CAST(solver AS VARCHAR(42)) = lower(address);
```
Comment (bh2smith) [inline models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql]: Thanks!

**Review by bh2smith — APPROVED** [review]
I am glad you kept the same file format as before. Changes look much better now. It should be fine to remove the old solvers (payout scripts do not ignore decommissioned/inactive solvers). Then your tests will pass.

**Comment by bh2smith** [comment]
Looks good and passing. Marking as Ready for review.

**Review by Hosuke — APPROVED** [review]

**Comment by gentrexha** [comment]
Hey @Hosuke, any ETA on when this will be merged?

**Comment by Hosuke** [comment]
> Hey @Hosuke, any ETA on when this will be merged?

Sorry, but please wait for a while, roughly about tomorrow.

**Review by jeff-dude — APPROVED** [review]
thank you for the patience team. will get this merged asap.

**Pull request merged**